helix-term/src/commands/typed.rs

```diff
@@ -785,6 +785,19 @@ fn reload(
     doc.reload(view.id)
 }
 
+fn reload_all(
+    cx: &mut compositor::Context,
+    _args: &[Cow<str>],
+    _event: PromptEvent,
+) -> anyhow::Result<()> {
+    for doc in cx.editor.documents_mut() {
+        doc.selections().keys().for_each(|view_id| {
+            doc.reload(*view_id);
```

Comment:
You shouldn't be calling `reload()` multiple times since it'll reload the file from disk on each call. Instead `reload` should be called with a single view (pick one at random)

Reply:
The reload method seems to be updating selections belonging to that view as well as history, does that not matter? I thought about refactoring the internal methods (apply, etc) to be view agnostic, but maybe they should loop through all views instead of doing it here?